src/web/lib/components/Modal/index.scss

```diff
@@ -39,9 +39,12 @@
   p {
     margin: 0;
   }
-
-  a:not(.modal__button) {
```

Comment:
Wouldn't just using `a:not(.modal__button):not(:visited)` have worked?

Reply:
a:not(.modal__button):not(:visited){
    color:var(--blue-50);
  }
it will not work @Rob--W

Reply:
That's interesting, because if I modify the CSS rule via the devtools of the browser, it does work: the visited link becomes purple.

How did you verify that "it will not work"?

Reply:
![Screenshot (56)](https://user-images.githubusercontent.com/58654018/91209462-69078f80-e729-11ea-8b2e-b606bca7c91b.png)
@Rob--W actually this is confusing me, It turns to purple color but I am not sure that after visiting it will turn to purple or at a first look it is a purple

Reply:
You can delete the entry from the browser history to see its normal color. Or open it in private browsing mode.

Reply:
a:not(.modal__button):not(:visited) {
    color: var(--blue-50);
  }
is I am writing correct code?

Reply:
Yes.

Reply:
then I will make pr of this code :)